olclient/cli.py

```diff
@@ -103,29 +107,29 @@ def main():
             raise
 
     if args.get_olid:
-        return ol.Edition.get_olid_by_isbn(args.isbn)
+        print(ol.Edition.get_olid_by_isbn(args.isbn))
     elif args.get_book:
         if args.olid:
-            return jsonpickle.encode(ol.Edition.get(olid=args.olid))
+            print(jsonpickle.encode(ol.Edition.get(olid=args.olid)))
         elif args.isbn:
-            return jsonpickle.encode(ol.Edition.get(isbn=args.isbn))
+            print(jsonpickle.encode(ol.Edition.get(isbn=args.isbn)))
     elif args.get_work:
         if args.olid:
-            return jsonpickle.encode(ol.Work.get(args.olid))
+            print(jsonpickle.encode(ol.Work.get(args.olid)))
         elif args.title:
-            return jsonpickle.encode(ol.Work.search(args.title))
+            print(jsonpickle.encode(ol.Work.search(args.title)))
     elif args.get_author_works:
         if args.olid:
-            return jsonpickle.encode(ol.Author.get(args.olid).works())
+            print(jsonpickle.encode(ol.Author.get(args.olid).works()))
         elif args.author_name:
-            return jsonpickle.encode(ol.Author.get(ol.Author.get_olid_by_name(args.author_name)).works())
+            print(jsonpickle.encode(ol.Author.get(ol.Author.get_olid_by_name(args.author_name)).works()))
     elif args.create:
         data = json.loads(args.create)
         title = data.pop('title')
         author = common.Author(data.pop('author'))
         book = common.Book(title, authors=[author], **data)
         edition = ol.Work.create(book)
-        return edition.olid
+        print(edition.olid)
     else:
         return parser.print_help()
```

Comment:
I don't think this needs a `return` -
`parser.print_help()` should be sufficient.

Reply:
I personally don't have a preference here. But I do agree `return` instead of `return None` would look simpler. 

Thanks for the feedback! Made changes @ https://github.com/internetarchive/openlibrary-client/pull/227/commits/1f3745f6f12f4ed22a30220cc02749024980e378